doc/experiment.rst

```diff
@@ -1,327 +1,423 @@
+is ideal for use cases where a SmartSim ``Model`` is run on a compute node
+that has hardware accelerators (e.g. GPUs) and low-latency inference is
+a critical component of the workflow.
+
+Colocated Deployment Diagram
+""""""""""""""""""""""""""""
+During colocated deployment, a SmartSim orchestrator (the database) runs on the same
+compute node(s) as a Smartsim model (the application).
+This type of deployment is optimal for high data inference scenarios.
+
+Below is an image illustrating communication
+between a colocated model spanning multiple compute nodes, and the database
+running on each application compute node. A single SmartRedis client is initialized
+for the colocated Orchestrator and is used to communicate with the application.
+Data is streamed from the application to the database via the client on the same node.
+
+.. figure:: images/co-located-orc-diagram.png
+
+Initialize a Colocated Orchestrator
+"""""""""""""""""""""""""""""""""""
+To create an ``Orchestrator`` that shares compute resources with a ``Model``
+SmartSim entity, use the ``model.colocate_db()`` factory method.
+In this case, the database
+is created via the SmartSim Model API function ``model.colocate_db``.
+The :ref:`Model API<model_api>` is accessed once a ``Model`` object has been initialized.
+
+
+Multi-db support
+^^^^^^^^^^^^^^^^
+SmartSim supports multi-database functionality, enabling an experiment
+to have several concurrently launched ``Orchestrator(s)``. If there is
+a need to launch more than one ``Orchestrator``, the ``Experiment.create_database()``
+function mandates the specification of a unique database identifier,
+denoted by the `db_identifier` argument, per created orchestrator.
+
+The `db-identifier` is used to reference SmartSim
+orchestrator(s) from application client code. This is particularly
+useful in instances where an orchestrator is colocated with a SmartSim
+model for low-latency inference and another Orchestrator is launched to
+handle other aspects of the workflow such as visualization and ML model
+training. More detailed information on the ideal use cases for orchestrator(s)
+and co-located ``Orchestrator(s)`` are available in sections... (update this when use cases added)
 
-In addition to a single model, SmartSim has the ability to launch an
-``Ensemble`` of ``Model`` applications simultaneously.
-
-An ``Ensemble`` can be constructed in three ways:
-  1. Parameter expansion (by specifying ``params`` and ``perm_strat`` argument)
-  2. Replica creation (by specifying ``replicas`` argument)
-  3. Manually (by adding created ``Model`` objects) if launching as a batch job
+Model
+-----
+``Model(s)`` represent any computational kernel, including applications,
+scripts, or generally, a program.
+They can interact with other
+SmartSim entities via data transmitted to/from SmartSim Orchestrators
+using a SmartRedis client.
+Models in PT, TF, and ONNX (scikit-learn, spark, and others) can be
```

Comment:
What you're describing here is more of what we refer to as a script rather than a Model. Even then, it's not python, but Torchscript